components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Chef Automate HA integrations"
+
+draft = false
+
+gh_repo = "automate"
+[menu]
+  [menu.automate]
+    title = "Integrations"
+    parent = "automate/deploy_high_availability"
+    identifier = "automate/deploy_high_availability/monitoring-alerting-centralized-logging-integration.md Chef Automate HA integrations"
+    weight = 75
++++
+
+You can integrate third-party platforms with [Chef Automate HA](/automate/ha) to monitor Autoamte's health, infrastructure, components, and services.
+
+Chef Automate HA integrates with the following platforms:
+
+- Datadog
+- AWS CloudWatch
+- Prometheus
+- Elastic Stack (ELK)
+
+## Datadog
+
+Datadog is a monitoring and analytics platform that delivers real-time insights into application, infrastructure, and cloud service performance. Refer to the following links for the detailed whitepapers and blog documents.
+
+- [Automate HA and DataDog Integration](https://github.com/chef/monitoring-integration-automate/tree/main/data-dog)
+- [Technical Blog reference of Automate HA and DataDog](https://progresssoftware-my.sharepoint.com/:w:/g/personal/apravati_progress_com/Eb2zXjNHa71Brszy3V_SqsoB0LtyGP7eaBXtBlaqj9rGSA?e=gJcC8b)
+
+## AWS CloudWatch
+
+Chef Automate HA with AWS CloudWatch gives automate and compliance teams the necessary redundancy and visibility to successfully operate the solution. Refer to the following links for the detailed whitepapers and blog documents.
+
+- [Automate HA and CloudWatch Integration](https://github.com/chef/monitoring-integration-automate/tree/main/cloud-watch)
+- [Technical Blog reference of Automate HA and CloudWatch](https://progresssoftware-my.sharepoint.com/:w:/g/personal/apravati_progress_com/EQsURaeVai5Eszc9-3yiHWEBKCeDV0Nz-LhqC2giqefO-Q?e=KM0iWB)
+
+## Prometheus
+
+Prometheus, an open-source tool excels at scraping metrics from HTTP endpoints, making it a go-to for harvesting data from various targets. Refer to the following links for the detailed whitepapers and blog documents.
+
+- [Automate HA and Prometheus Integration](https://github.com/chef/monitoring-integration-automate/tree/main/prometheus)
+- [Technical Blog reference of Automate HA and Prometheus](https://progresssoftware-my.sharepoint.com/:w:/g/personal/apravati_progress_com/EZ23D4XeuKJGu-yMzmIehfwBPmTouhykSFqRhCJXdWSQWg?e=tysXun)
+
+## Elastic Stack
+
+The ELK stack is a trio of open-source tools that have revolutionized how businesses harness their data. Refer to the following links for the detailed whitepapers and blog documents.
+
+- [Automate HA and ELK Integration](https://github.com/chef/monitoring-integration-automate/tree/main/ELK)
```

Comment:
@dishanktiwari2501 please update the data dog whitepaer link with whitepaper data dog section link, which is:
https://github.com/chef/monitoring-integration-automate/blob/main/Whitepaper_AutomateHA_Monitoring_and_Alerting.md#elk-integration-with-automate-ha---centralised-logging